tests/translator/test_translator.py

```diff
@@ -339,8 +372,8 @@ def _update_logical_id_hash(self, resources):
 
         # Update any Output References in the template
         for output_key, output_value in resources.get("Outputs", {}).items():
-            if output_value.get("Ref") in deployment_logical_id_dict:
-                output_value["Ref"] = deployment_logical_id_dict[output_value.get("Ref")]
+            if output_value.get("Value").get("Ref") in deployment_logical_id_dict:
+                output_value["Value"]["Ref"] = deployment_logical_id_dict[output_value.get("Value").get("Ref")]
```

Comment:
The code was (incorrectly) trying to parse outputs in the following manner:

```
Outputs:
  MyOutput: !Ref Deployment34567
```

I updated it (and the tests) to be correct:

```
Outputs:
  MyOutput:
    Value: !Ref Deployment34567
```